src/backend/aspen/api/utils/phylo.py

```diff
@@ -134,47 +186,74 @@ async def _set_countries(
         .select_from(origin_location)  # type: ignore
         .join(
             Location,  # type: ignore
-            and_(
-                Location.division == None,  # noqa: E711
-                Location.location == None,  # noqa: E711
-                Location.country.in_(defined_countries),
-            ),
+            or_(*and_clauses),
         )
         .where(
             and_(
                 origin_location.country == tree_location.country,
-                origin_location.division == None,  # noqa: E711
-                origin_location.location == None,  # noqa: E711
+                origin_location.division == tree_location.division,
+                origin_location.location == tree_location.location,
             )
         )
         .order_by(asc("distance"))
-        .limit(15)
+        .limit(16)
```

Comment:
we're pulling 16 closest directly from the database which will include the `tree_location` itself, so we don't need to add into the `tree_location` later.